src/main/msp/msp.c

```diff
@@ -643,8 +641,15 @@ static bool mspCommonProcessOutCommand(int16_t cmdMSP, sbuf_t *dst, mspPostProce
         sbufWriteU8(dst, FC_VERSION_PATCH_LEVEL);
         break;
 
-    case MSP_BOARD_INFO:
-    {
+    case MSP2_MCU_TYPE_INFO: {
```

Comment:
Maybe `MSP2_MCU_INFO` and add other MCU info in future (clock, memory, UID, ... )